azure-pipelines-official.yml

```diff
@@ -37,9 +37,16 @@ stages:
     - powershell: Write-Host "##vso[task.setvariable variable=SourceBranchName]$('$(Build.SourceBranch)'.Substring('refs/heads/'.Length))"
       displayName: Setting SourceBranchName variable
 
+    - powershell: Write-Host "##vso[task.setvariable variable=NonVSDepsBranch]$($Build.SourceBranch -like '*-vs-deps')"
+      displayName: Setting VSDepsBranch variable
+
     - powershell: Write-Host "##vso[task.setvariable variable=VisualStudio.DropName]Products/$(System.TeamProject)/$(Build.Repository.Name)/$(SourceBranchName)/$(Build.BuildNumber)"
       displayName: Setting VisualStudio.DropName variable
 
+    - powershell: git branch -r | select-string "origin/$(Build.SourceBranch)-vs-deps" | select -First 1 | %{ Write-Host "git merge $_" }
```

Comment:
This means we can't do official builds on non-vs-deps branches if they don't have a -vs-deps branch? Probably shouldn't error in this case. Or at least add a condition that allows it to be overridden.

Reply:
Well, it means if it doesn't exist the `git merge` won't run. Maybe I could make that more clear. It's like enumerating over an empty list, so it doesn't happen